webapp/app/utils/regtest.ts

```diff
@@ -0,0 +1,13 @@
+import { defineChain } from "viem";
+
+export const regtest = defineChain({
+  id: 11155111,
+  name: "Sepolia",
+  rpcUrls: {
+    default: {
+      http: ["http://localhost:8545"],
+    },
+  },
+  nativeCurrency: { name: "Sepolia Ether", symbol: "ETH", decimals: 18 },
+  testnet: true,
+});
```

Comment:
We needed to have a network defined exactly as sepolia but using the local rpc. I hardcoded the localhost default url to avoid sending the rpc url to the front, because in production that should not be done.

Maybe there is a more standard way to do this. Open to do any improvements here.